### ISSUE TITLE
NTR: Modifier: nonexudative character to conjunctivitis (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7994]
New term request Modifier: nonexudative character to conjunctivitis (seidel_190131164321): Conjunctiva

**Comment by pnrobinson** [comment]
@MickeySegal
is this for Kawasaki?
Possibly it would be better to have a term "Nonexudative conjunctivitis" since I am not sure we would use the modifier otherwise?

**Comment by MickeySegal** [comment]
This is for a variety of conditions such as allergic conjunctivitis, mucous membrane pemphigoid molluscum contagiosum and Kawasaki.  

Nonexudative conjunctivitis is fine.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000843
- label: Nonexudative conjunctivitis
- parents: [Conjunctivitis (HP:0000509)](https://hpo.jax.org/app/browse/term/HP:0000509)
- pmid: 
- def: A type of conjunctivitis that is not characterized by appreciable exudate, i.e., fluid, cells, or other substances that have escaped from blood vessels and have been deposited around the eyelids.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7994
- synonyms:
